.github/workflows/mediasoup-pack.yaml

```diff
@@ -0,0 +1,125 @@
+name: mediasoup-pack
+
+on:
+  push:
+    tags:
+    - v*  # Only build for tags that match new versions of `npm` packages
+  # Manual dispatch, only for testing purposses
+  # See https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+
+jobs:
```

Comment:
These will not generally work unless you're on the same version of the same OS. Worker binary is not static in mediasoup. Unless you make it such, this approach will generate a lot of bug reports.

Reply:
Could the solution of using the oldest OS work? What can be done regarding the different C libraries? I think there's no way to detect that on Node.js, although static builds (real ones, not glibc ones, that always has some dynamic dependencies on it) could work.

Reply:
I have seen binaries built on Windows Server 2019 (roughly Windows 10) not working on Windows 11, so you'll have to provide proof that it will work without issues.

Reply:
So, how can we do that? Maybe using the oldest libc version is not enough? In theory, Windows is fully backwards binary compatible, in some cases for decades...